src/app/core/create-store.js

```diff
@@ -0,0 +1,29 @@
+
+/**
+ * Dependencies.
+ */
+
+var createStore = require('fluxible/utils/createStore');
+var extend = require('extend');
+
+/**
+ * Generic hydrate and dehydrate methods.
+ */
+
+var genericHydration = {
+  rehydrate: function (state) {
+    this.state = state;
+  },
+  dehydrate: function () {
+    return this.state;
+  }
+};
+
+/**
+ * An extension of fluxible's createStore method.
+ */
+
+module.exports = function (storeSpec) {
+  extend(false, storeSpec, genericHydration);
+  return createStore(storeSpec);
+};
```

Comment:
Watch out for ending lines. I think git will show this sort of thing in the diff (which is worth reviewing before committing anyway).